src/main/java/frc/robot/RobotContainer.java

```diff
@@ -156,6 +156,10 @@ private void configureBindings() {
                   elevator.setNeutralMode(NeutralModeValue.Coast);
                 })
             .ignoringDisable(true));
+    Commands.runOnce(
```

Comment:
This does not actually run the command.   It creates a command that can be run, if attached to a trigger or condition.
Here it just dies.

Reply:
It should be `IS_DISABLED.onTrue(driveCommand.stopDrivetrain());`